framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -113,7 +109,8 @@ class SimulatorFragmentRenderingRules(object):
     def __init__(self, form=None, form_action_url=None, previous_form_action_url=None, is_simulation_copy=False,
                  is_simulation_readonly_load=False, last_form_url=SimulatorWizzardURLs.SET_CONNECTIVITY_URL,
                  last_request_type='GET', is_first_fragment=False, is_launch_fragment=False, is_model_fragment=False,
-                 is_surface_simulation=False, is_noise_fragment=False, is_launch_pse_fragment=False, is_pse_launch=False):
+                 is_surface_simulation=False, is_noise_fragment=False, is_launch_pse_fragment=False, is_pse_launch=False,
+                 is_monitor_params_fragment=False, monitor_name=None):
```

Comment:
If the only purpose of flag is_monitor_params_fragment is to display the monitor_name in the template, then we can remove it, and check if monito_name is not None directly.